charts/osm/templates/osm-gateway-deployment.yaml

```diff
@@ -54,4 +55,15 @@ spec:
         - name: envoy-bootstrap-config-volume
           secret:
             secretName: osm-gateway-bootstrap-config
-{{- end }}
\ No newline at end of file
+
+---
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    {{- include "osm.labels" . | nindent 4 }}
+    app: osm-multicluster-gateway
+  name: osm-multicluster-gateway
+  namespace: {{ include "osm.namespace" . }}
+{{- end }}
```

Comment:
@draychev it would be helpful to note in the commit message description why a separate service account is required.